ansible/roles/postgresql/templates/rotate_logs.sh.j2

```diff
@@ -1,6 +1,9 @@
 #!/bin/bash
 # keep latest {{ postgresql_num_logs_to_keep }} log files
-find {{ postgresql_log_directory }} -type f -name '*.csv' -printf '%T@\t%p\n' \
+find {{ postgresql_log_directory }} \
+    -type f \
+    \( -name '*.csv' -o -name '*.csv.gz' -o -name '*.log' \) \
```

Comment:
nit: This is getting pretty nasty. maybe we should break these up, or use a for loop for them

Reply:
I'm not sure how we'd do this with a for loop. Feel free to improve it.